mrp_mto_with_stock/models/mrp_production.py

```diff
@@ -0,0 +1,114 @@
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
+# Copyright 2015 John Walsh
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, models
+from odoo.exceptions import UserError
+import logging
+_logger = logging.getLogger(__name__)
+
+
+class MrpProduction(models.Model):
+    _inherit = 'mrp.production'
+
+    @api.multi
+    def _adjust_procure_method(self):
+        # Si location => By pass method...
+        super(MrpProduction, self)._adjust_procure_method()
```

Comment:
The code is the same in the 10.0 version, but I don't see the point of it

Reply:
Yes. I am agree. We will remove it.

Reply:
Corresponded. Thank you.